### PR TITLE
[PLT-0] Prep Release v.3.75.0

### PR DESCRIPTION
**Pull request by adrian-chang**
# Description

Release v.3.75.0

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Document change (fix typo or modifying any markdown files, code comments or anything in the examples folder only)

## All Submissions

- [ ] Have you followed the guidelines in our Contributing document?
- [x] Have you provided a description?
- [x] Are your changes properly formatted?

### CODE REVIEW

**Review by tomislav-peharda — DISMISSED** [review]

File: libs/labelbox/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,28 @@
 # Changelog
+# Version 3.75.0 (2024-07-10)
+
+## What's Changed
+* [PLT-0] Predictions missing by @adrian-chang in https://github.com/Labelbox/labelbox-python/pull/1695
+* Use correct project type in tests/integration/test_project_model_config.py by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1697
```
Comment (vbrodsky) [inline libs/labelbox/CHANGELOG.md]: do we need to add tests stuff? I usually do not

File: libs/labelbox/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,28 @@
...
+* [PLT-1151] Vb/model setup complete by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1685
```
Comment (vbrodsky) [inline libs/labelbox/CHANGELOG.md]: Looks like these are generated from PR comments... should we have it a little more polished? I usually do

**Review by vbrodsky — COMMENTED** [review]
My comments are re CHANGELOG messages
I've been following the feedback previously received from MLSE (John?), which involved reviewing and polishing merge commit messages to ensure a professional look in the CHANGELOG. Additionally, I've omitted comments related to internal commits focused solely on test fixes and refactoring.
However, I'm unsure if this approach is still the preferred method.  Has there been a shift towards including all commits in the CHANGELOG, regardless of their nature?
Even we stick with this approach, I think we need to fix text for a couple of those commits messages

File: libs/labelbox/CHANGELOG.md (hunk 1 above)
Comment (adrian-chang) [reply libs/labelbox/CHANGELOG.md]: (Addressing your feedback also above), 

I've already edited the autogenerated notes (given it a pass). I'll remove this one in particular.

File: libs/labelbox/CHANGELOG.md (hunk 2 above)
Comment (adrian-chang) [reply libs/labelbox/CHANGELOG.md]: They are, and I'm trying to minimize the time spent on changelog notes. Some of it is useful (the URLS) although some of it is not so useful (random / odd PR titles).

File: libs/labelbox/CHANGELOG.md [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
-
-## What's Changed
-* [PLT-0] Predictions missing by @adrian-chang in https://github.com/Labelbox/labelbox-python/pull/1695
-* Use correct project type in tests/integration/test_project_model_config.py by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1697
-* [PLT-1151] Vb/model setup complete by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1685
-* [PLT-1205] Add user group management by @adrian-chang in https://github.com/Labelbox/labelbox-python/pull/1604
-* [PLT-1205] Fix integration tests by @adrian-chang in https://github.com/Labelbox/labelbox-python/pull/1698
-* [PTDT-2410] Prevent adding batches to live chat evaluation projects by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1703
+## Added
+* Project Model Setup by @vbrodsky in https://github.com/Labelbox/labelbox-python/pull/1685
```
Comment (vbrodsky) [inline libs/labelbox/CHANGELOG.md]: I would say
Add Project set_project_model_setup_complete() method

**Review by vbrodsky — DISMISSED** [review]

**Review by sfendell-labelbox — APPROVED** [review]

**Pull request merged**